addons/calendar/views/calendar_views.xml

```diff
@@ -194,7 +194,6 @@
                                 <group>
                                     <field name="privacy"/>
                                     <field name="show_as"/>
-                                    <field name="rrule" invisible="1" readonly="0" />
```

Comment:
Why removing this?

Reply:
Should be mentioned in the commit message...
As Jem told me, this field is essentially useless, and if it did anything at all it could only interfere with the logic of the rrule creation, so it'd be better to remove it altogether.